src/structures/interfaces/InteractionResponses.js

```diff
@@ -14,12 +14,20 @@ class InteractionResponses {
    * Options for deferring the reply to an {@link Interaction}.
    * @typedef {Object} InteractionDeferOptions
    * @property {boolean} [ephemeral] Whether the reply should be ephemeral
+   * @property {boolean} [fetchReply] Whether to return the replied interaction
```

Comment:
Shouldn't this specify that it fetches the reply if possible? @discordjs/the-big-3

Reply:
I personally wasn't sure what to put here, so any suggestions are welcome! Although now that you say that, "Whether to fetch the reply" could be an option!